vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -1692,6 +1823,52 @@ func flattenVsanDisks(d *schema.ResourceData, cluster *object.ClusterComputeReso
 	return d.Set("vsan_disk_group", diskMap)
 }
 
+func flattenVsanStretchedCluster(client *vsan.Client, d *schema.ResourceData, cluster *object.ClusterComputeResource, obj *types.ClusterConfigInfoEx) error {
+	res, err := vsanclient.GetWitnessHosts(client, cluster.Reference())
+	if err != nil {
+		return err
+	}
+
+	if res.Returnval == nil {
+		return d.Set("vsan_stretched_cluster_enabled", false)
+	}
+
+	if res.Returnval[0].UnicastAgentAddr != "" {
+		conf := []interface{}{}
```

Comment:
you don't need to initilized conf, using `append` will handle that for you:
```suggestion
		var conf []interface{}
```